src/optiondefs.h

```diff
@@ -2021,6 +2021,9 @@ static struct vimoption options[] =
     {"pumheight",   "ph",   P_NUM|P_VI_DEF,
 			    (char_u *)&p_ph, PV_NONE, NULL, NULL,
 			    {(char_u *)0L, (char_u *)0L} SCTX_INIT},
+    {"pummaxwidth", "pmw",   P_NUM|P_VI_DEF,
+			    (char_u *)&p_pmw, PV_NONE, NULL, NULL,
```

Comment:
```suggestion
			    (char_u *)&p_pmw, PV_NONE, did_set_pummaxwidth, NULL,
```
And then define a `did_set_pummaxwidth()` function to validate, that it is always positive.

Reply:
I saw that pumwidth doesn't validate for positive numbers so I skipped it

Reply:
We basically don't have did_set and test for integer options because we always check if it is > 0 when using it and I don't think this condition can be skipped by validation, because this option only works if it is greater than 0..

Reply:
Ah I see. Okay